config-schema.json

```diff
@@ -752,27 +785,28 @@
         "proximity": {
           "$id": "#/properties/settings/properties/proximity",
           "type": "integer",
-          "minimum": 0,
           "title": "Proximity sensitivity",
-          "default": 40
+          "default": 40,
+          "maximum": 255,
+          "minimum": 10
         },
         "shutdowntime": {
           "$id": "#/properties/settings/properties/shutdowntime",
           "type": "integer",
-          "title": "Shutdown time in minutes",
-          "default": 21600
+          "title": "Shutdown time in seconds",
+          "default": 0
         },
         "softwareupdate": {
           "$id": "#/properties/settings/properties/softwareupdate",
           "type": "boolean",
-          "title": "Enable software updates",
+          "title": "Enable automatic software updates",
           "default": false
         },
         "wifitime": {
           "$id": "#/properties/settings/properties/wifitime",
           "type": "integer",
-          "title": "WiFi switch off time in minutes",
-          "default": 3600
+          "title": "WiFi switch off time in seconds",
+          "default": 0
```

Comment:
Shouldn't we switch off wifi by default? Of course we need to keep it on during the initial setup.
Why seconds?
I personally find minutes more convenient in a configuration file. No need to specify seconds.

Reply:
This is never shown like this to the user. `config.json` is only edited by us for now.